modules/social_features/social_topic/config/modify/social_topic.authenticated_taxonomy_permissions.yml

```diff
@@ -0,0 +1,13 @@
+dependencies:
+  module:
+    - taxonomy_access_fix
```

Comment:
Does this means that config modify will check for existence of module first? Or, just add these properties to current configuration?

Reply:
@navneet0693 This means the permission only will be allowed if the Taxonomy Access Fix module is enabled.

Reply:
@vcsvinicius Thank you for educating me!